Resources/doc/features/tagging.rst

```diff
@@ -32,10 +32,10 @@ For more information, see :doc:`/reference/configuration/tags`.
 Setting and Invalidating Tags
 -----------------------------
 
-You can tag responses in three ways: with the cache manager, configuration and
-annotations.
+You can tag responses in three ways: with the tag handler, from configuration
+and using annotations.
```

Comment:
This is a bit weird: you can tag in three ways, but Twig is not mentioned here but only below.